### ISSUE TITLE
geobound required parameters?

### ISSUE THREAD

**Issue by smulube** [HyperCatIoT/hc2-server#2]
I was interested to see testcases for geobound search requests where just a single geobound parameter is supplied, e.g.

``` javascript
it("should work with geobound-minlat only", function() {
       return chai.request(settings.getBaseURL())
            .get(catUrl + '&geobound-minlat=50.1242')
            .then(function(res) {
                res.body.should.have.property('items').which.has.length(1);
            });
    });
```

However there was this line in the specification document:

> Where a geographic search request is performed, the client shall include the query parameters set out in Table 14

I'd taken this to mean that for a geobound search all four parameters were _required_ (i.e. `geobound-minlong`, `geobound-maxlong`, `geobound-minlat` and `geobound-maxlat`), and the server should return an error of some sort?

**Comment by tobyjaffey** [comment]
Yes, I agree. A geobound search without all parameters should be considered invalid. We need a test for this.
